RUNNING_IN_CONTAINERS.md

```diff
@@ -0,0 +1,62 @@
+# Running in Containers
+In case you want to run radicle in containers, on the same host (e.g. your laptop),
+you can use the `docker-compose.yml` file provided within this repo. 
+
+For the first node you want to start on your machine, you can: 
+
+## Create a profile 
+1. Create a folder where you will store the data of your node. e.g. `mkdir -p ~/radicle/profiles/bob/.radicle`
+1. Set `RAD_HOME` : `export RAD_HOME=~/radicle/profiles/bob/.radicle`
+1. Create a key:
+   - Pick a good passphrase and store it in your password manager
+   - go ahead with creating the key `rad auth --stdin` (or use `RAD_PASSPHRASE` env var)
+   - your profile should be created in `~/radicle/profiles/bob/.radicle`. 
+
+## Build the container images
+
+This takes a couple of minutes - depending on your machine - as it needs 
+to download the parent container images, and also all the rust 
+dependencies and then compile the code: 
+
+```bash
+docker-compose build
```

Comment:
Or we could open up our GCP containers repository to be public to make this easier.  It's got pre-built images that can be simply downloaded and run.  CC @cloudhead